README.md

```diff
@@ -18,6 +18,8 @@ This project aims to contrast and compare multiple low-cost metrics for selectin
    pip install .
    ```
 
+3. (temporary) if using LogME, NCE or LEEP fork the following repository <https://github.com/thuml/LogME> and clone into `src`.
```

Comment:
The fork step isn't necessary here (only necessary to make changes to the LogME code, otherwise just clone is fine). But we should mention which commit we have tested locomoset on (https://github.com/thuml/LogME/commit/c79823694a8bfa4292a76277cb1ac2cdec89022f)

Maybe we should also add a references section to the readme and include both [LogME](https://github.com/thuml/LogME#citation) and [PARC](https://github.com/dbolya/parc#citation) citations

Reply:
Updated the README